### PR TITLE
Fix for spacing background colour

### LINKED ISSUE #1542: Footer top padding doesnt account for ghost wrap
**Describe the bug**
The footer has a top margin, which is pushing the ghost wrapper up, causing a white gap

**To Reproduce**

1. Go to any article
2. See error

**Expected behavior**
The padding incs ghost wrap

**Screenshots**
Screenshot replacing ghost with red

image

**Additional context**

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #1542

**Overall change:** Fixes issue to do with gap at bottom being in white instead of `Ghost`.

**Code changes:**

- Remove the padding top from the footer and add it as a padding bottom to the grid
- Snapshots updated

No regression should be seen on Front pages. 
On Articles, spacing between the content and the footer should be identical, just that the spacing should be coloured `Ghost` and not the default browser white. 

Screen Shot 2019-07-18 at 10 58 17

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by sareh** [comment]
Has been UX reviewed by @nataliesmart

File: src/app/lib/styledGrid/index.jsx [hunk 1]
```diff
@@ -21,11 +22,20 @@ export const Grid = styled.div`
   ${layoutGridWrapper};
 `;
 
+export const GridWithPadding = styled.div`
+  ${layoutGridWrapper};
+  padding-bottom: ${GEL_SPACING_QUAD};
+`;
+
 export const GhostGrid = styled.div`
```
Comment (dr3) [inline src/app/lib/styledGrid/index.jsx:26]: Does anything still use these?

Comment (sareh) [reply src/app/lib/styledGrid/index.jsx:26]: Nothing in production, only storybook stories. We will likely need `Grid` for nested grids shortly.

Comment (sareh) [reply src/app/lib/styledGrid/index.jsx:26]: Implementation changed to add the padding directly onto the grid.

**Review by dr3 — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Checked this for article pages:

    http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po
    http://localhost.bbc.com:7080/persian/articles/c4vlle3q337o

The spacing for background color is now fixed. Can be merged.

**Pull request merged**